home-data-migration/pom.xml

```diff
@@ -32,12 +31,7 @@
                         <artifactId>liquibase-maven-plugin</artifactId>
                         <version>${liquibase.version}</version>
                         <configuration>
-                            <promptOnNonLocalDatabase>false</promptOnNonLocalDatabase>
-                            <changeLogFile>db/changelog/master.xml</changeLogFile>
-                            <driver>org.postgresql.Driver</driver>
-                            <url>jdbc:postgresql://localhost:5433/postgres</url>
-                            <username>postgres</username>
-                            <password>password</password>
+                            <propertyFile>config/liquibase.properties</propertyFile>
```

Comment:
I don't have a strong opinion on it but I think keeping these properties in maven config much preferable. Just move it to the properties section so anybody can override it: `mvn clean install -Dproperty.to.override`

Reply:
Done.